docs/remote/containers-advanced.md

```diff
@@ -537,25 +537,30 @@ Once the needed CLIs are in place, you can also work with the appropriate contai
+* [Docker-from-Docker](https://aka.ms/vscode-remote/samples/docker-from-docker) - Also known as "Docker-outside-of-Docker", this illustrates how you can use the Docker (or Moby) CLI in your dev container to connect to your host's Docker daemon by bind mounting the Docker Unix socket. Lower overhead and can reuse your machine's cache, but has [bind mounting limitations](#mounting-host-volumes-with-docker-from-inside-a-container).
 
-* [Kubernetes-Helm](https://aka.ms/vscode-remote/samples/kubernetes-helm) - Includes the Docker CLI, kubectl, and Helm and illustrates how you can use them from inside a dev container to access a local Minikube or Docker provided Kubernetes cluster.
+* [Docker-from-Docker Compose](https://aka.ms/vscode-remote/samples/docker-from-docker-compose) - Variation of Docker-from-Docker for situations where you are using Docker Compose instead of a single Dockerfile.
 
-Note that it is possible to actually run the Docker daemon inside a container. While using the [`docker` image](https://hub.docker.com/_/docker) as a base for your container is an easy way to do this, given [the downsides](https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/) and performance implications, using the Docker CLI to access your local Docker host from inside a container is typically a better option.
+* [Kubernetes-Helm](https://aka.ms/vscode-remote/samples/kubernetes-helm) - Takes the Docker-from-Docker model and adds kubectl and Helm to illustrate how you can access a local Minikube or Docker provided Kubernetes cluster.
 
 ### Mounting host volumes with Docker from inside a container
 
-When using the Docker CLI from inside a container, the host's Docker daemon is used. This effects mounting directories from inside the container as the path inside the container may not match the path of the directory on the host.
+When following the [Docker-in-Docker](https://aka.ms/vscode-remote/samples/docker-in-docker) model, using the Docker CLI from inside a dev container will cause it to interact with a Docker daemon running in the same place. This means that you can "bind" mount anything inside the dev container into the "inner" containers you create.
 
-For example:
+For example, this will "just work":
 
 ```bash
-docker run -v /workspace/examplefile.txt:/incontainer/path busybox
+docker run -v /workspace/examplefile.txt:/incontainer/path debian
 ```
+However, if you wish to make a host folder available to this inner container, you need to [mount it](#adding-another-local-file-mount) into your dev container first.
+
+With [Docker-from-Docker](https://aka.ms/vscode-remote/samples/docker-from-docker), the situation is reversed. Here, the Docker CLI inside the container interacts with the host's Docker daemon instead. This effects mounting directories from inside the container as the path inside the container may not match the path of the directory on the host.
```

Comment:
The use of "reversed" was a bit confusing to me at first. From my understanding, Docker-in-Docker has the child containers all within the dev container (independent from the host), while Docker-from-Docker accesses the host's Docker daemon. Is it a "reverse" because containers are inside the dev container in one case, and outside in the other?

Reply:
For docker-in-docker, you can access all folders within the container, but not the host.  In docker-from-docker, you can access the host but not the containers folders. But its interesting it doesn't read that way.

Took a crack at clarifying.